### DISCUSSION TITLE
title bar for one window

### DISCUSSION THREAD

**Discussion by qrockz** [i3/i3#6031] [Q&A] [answered]
When I'm in stacked or tabbed mode and have multiple windows opened in one workspace then the title bar is activated for each window.
I want this behaviour also when only one window is opened in the workspace. Is this possible?

**Comment by orestisfl** [comment]
Not sure if I understand the question correctly but that's the default behavior.

**Reply by qrockz** [reply]
Basically I want the title bar also be visible if only one window is visible. Currently the title bar shows up only when at least 2 windows are present. When only one window is present then there is no title bar - thats very :cry:

**Reply by orestisfl** [reply]
This is the default behavior, share your config please.

**Reply by qrockz** [reply]
[config.txt](https://github.com/i3/i3/files/15164052/config.txt)

**Reply by orestisfl** [reply]
```
default_border pixel 1
default_floating_border pixel 1
for_window [class="^.*"] border pixel 1
```

This lines in your config set your border to "pixel" which disables the title bar.

**Reply by qrockz** [reply]
[screen recorded](https://github.com/i3/i3/assets/56853194/54211f6a-f41e-46c1-9434-b59007e4580f)

increased the border size but window title is not available when only one window is present.
```
default_border pixel 20
default_floating_border pixel 20
# for_window [class="^.*"] border pixel 1
```

**Reply by orestisfl** [reply, marked as answer]
The `pixel` part is the problem, you need to use `normal` instead. See the user guide for reference.
